api/external/sms.html

```diff
@@ -0,0 +1,376 @@
+<!doctype html>
+
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+    <title>Emma API Documentation | Emma, Inc.</title>
+
+
+    <link rel="stylesheet" href="../../_static/emma.css" type="text/css" />
+    <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
+
+    <script type="text/javascript">
+        var DOCUMENTATION_OPTIONS = {
+            URL_ROOT:    '../../',
+            VERSION:     '0.1',
+            COLLAPSE_INDEX: false,
+            FILE_SUFFIX: '.html',
+            HAS_SOURCE:  true
+        };
+    </script>
+
+    <script src="https://code.jquery.com/jquery-1.4.min.js"></script>
+    <script>window.jQuery || document.write('<script src="_static/jquery.js"><\/script>')</script>
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+    <script type="text/javascript" src="../../_static/underscore.js"></script>
+    <script type="text/javascript" src="../../_static/doctools.js"></script>
+    <link rel="top" title="audience 0.1 documentation" href="../../index.html" />
+    <link rel="next" title="Webhooks" href="../../webhooks.html" />
+    <link rel="prev" title="Subscriptions" href="subscriptions.html" />
```

Comment:
are we wanting to line these up with the other changes you made to subscriptions and signup_forms?

Reply:
That's a good point. It looks like we'd need to update these similar links in [the other templates](https://github.com/myemma/emma-api-documentation/blob/8438cf978505da94c63eb2ea01c733b2adf35c6d/api/external/signup_forms.html#L32-L33) too

Reply:
I guess we'd only need to update Webhooks, Subscriptions looks correct here

```suggestion
    <link rel="next" title="Signup Forms" href="signup_forms.html" />
    <link rel="prev" title="Subscriptions" href="subscriptions.html" />
```

Reply:
dang I missed one lol